modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/http/access/AccessLogger.java

```diff
@@ -211,6 +254,29 @@ protected synchronized void open() {
         }
     }
 
+    /*
+        When some error happens, create a file with yesterday timestamp
+     */
+    private void getInErrorFileName(File dir, File existing) {
```

Comment:
the method name is not descriptive of what it does, shall we correct it?

Reply:
+1 filed